### ISSUE TITLE
Wrong page opens on long reference strips

### ISSUE THREAD

**Issue by adversor** [openseadragon/openseadragon#1992]
It was fine in v2.4.2, only an issue with new source.

The reference strip page call gets confused with longer reference strips (>10 to 20 images). It seems the used panelWidth for page calculation is 4px too small. I could get it to run by adding 4 to panelWidth in the page calculation in referencestrip.js. Reducing panelWidth in line 178 where 8 is added will mess up the total length of the reference strip: 

```
function onStripClick( event ) {
    if ( event.quick ) {
        var page;
        if ( 'horizontal' === this.scroll ) {
            page = Math.floor(event.position.x / (this.panelWidth + 4));
        } else {
            page = Math.floor(event.position.y / this.panelHeight);
        }

        this.viewer.goToPage( page );
    }

    this.element.focus();
}

    this.element.focus();
}
```

**Comment by msalsbery** [comment]
Thank you for catching this! Will be fixed for v3.0.0

**Comment by adversor** [comment]
@msalsbery Great! Thank you.

**Comment by msalsbery** [comment]
@adversor  For what it's worth...if you need it right away [here's the commit that fixes this hopefully](https://github.com/openseadragon/openseadragon/pull/1961/commits/7d19edff34f7491248064bef1efa69af588c40a0)

The final version when #1961 lands may be different, but the fix will be there as well

**Comment by adversor** [comment]
@msalsbery Thank you for taking care. I will update to 3.0.0 when it is available, until now it works with my dirty "+4" fix.

**Comment by damonsson** [comment]
Looks like fix from that draft commit not solve problem. @adversor solution do that. If it is still not fixed by 2 years, maybe better is to use that dirty fix from adversor, than wait for ideal solution? Will you accept PR with that dirty solution?

**Comment by iangilman** [comment]
@damonsson Yeah, seems reasonable. Perhaps with a comment explaining the use of the magic number. Would you be up for making that PR?

**Comment by damonsson** [comment]
> @damonsson Yeah, seems reasonable. Perhaps with a comment explaining the use of the magic number. Would you be up for making that PR?

Done ;)

**Comment by iangilman** [comment]
Fixed by #2280.
